src/js/views/patients/patient/flow/patient-flow.scss

```diff
@@ -106,3 +106,33 @@
   padding: 20px;
   text-align: center;
 }
+
+.patient-flow__header__progress {
+  background: #E6E6E6;
+  border: 0;
+  border-radius: 16px;
+  display: inline-block;
+  height: 16px;
+  margin-top: 0.75em;
+  width: 144px;
+}
+
+.patient-flow__header__progress::-webkit-progress-bar {
+  background: transparent;
+}
+
+.patient-flow__header__progress::-moz-progress-bar {
+  background: #1dbe6c;
```

Comment:
this also should likely be one of our green colors in sass and not hard-coded here.